CMakeLists.txt

```diff
@@ -55,14 +57,15 @@ if(EVMONE_FUZZING)
         message(FATAL_ERROR "Clang 8+ compiler is required for fuzzing")
     endif()
 
-    option(EVMONE_FUZZING_COVERAGE "Fuzzing coverage" OFF)
-
     set(BUILD_SHARED_LIBS OFF CACHE BOOL "Build evmone as a shared library" FORCE)
     set(EVMONE_TESTING ON CACHE BOOL "Build tests and test tools" FORCE)
 
-    if(EVMONE_FUZZING_COVERAGE)
-        add_compile_options(-fprofile-instr-generate -fcoverage-mapping)
```

Comment:
These are never added now? why?

Reply:
These are added by `-DCMAKE_BUILD_TYPE=Coverage`.

Reply:
Is it supported directly by cmake?

Reply:
Cmake supports custom build types. Here the "Coverage" build type is defined (#176) and it adds the mentioned flags.

Reply:
Ah I think I see, cmake gets flags for the custom build type from the variable containing that build type name, here `CMAKE_CXX_FLAGS_COVERAGE`